### ISSUE TITLE
MMT surface syntax highlighting on http://gl.mathhub.info

### ISSUE THREAD

**Issue by kohlhase** [MathHubInfo/Frontend#49]
Now that we have source links to GitLab (see #41), it would be very nice to have syntax higlighting for the MMT files there. It seems that GitLab uses [rouge](http://rouge.jneen.net/) and we can configure GitLab to use special syntax highlighting for `*.mmt` via https://docs.gitlab.com/ee/user/project/highlighting.html; I am not sure how to install a new syntax highlighter yet.

**Comment by kohlhase** [comment]
Made a corresponding [MMT issue](https://github.com/UniFormal/MMT/issues/410), this issue here should concentrate on GitLab config issues  once that is done (or started).

**Comment by tkw1536** [comment]
This was done some time ago
